test/fixtures/profiles/profile-with-comments-and-empty-file/controls/example_with_only_comments.rb

```diff
@@ -0,0 +1,13 @@
+# # copyright: 2018, The Authors
+
+# title "sample section"
```

Comment:
Is it possible for you to separate these 3 different test case scenarios into 3 different test profiles? You might actually not need `test/fixtures/profiles/profile-with-comments-and-empty-file/controls/example.rb` this then. So only 2 profiles then. Usually, we also recommend having different test scenarios in different profiles.

Reply:
My thought process was that the profile contains multiple files in the controls directory - of which some are empty, some have a basic control and some only have comments. Because of this, I decided to keep it as a single profile instead of splitting it into multiple profiles.